### DISCUSSION TITLE
Web UI for Whisper API

### DISCUSSION THREAD

**Discussion by felixbade** [openai/whisper#1018] [Show and tell]
I made a simple front-end for Whisper, using the new API that OpenAI published. I hope this lowers the barrier for testing Whisper for the first time.

The web page makes requests directly to OpenAI's API, and I don't have any kind of server-side processing myself. Feel free to use this tool for whatever – either through this page, or by running your own instance.

https://transcribe.bloat.app

GitHub: https://github.com/felixbade/transcribe

I would love to add diarization ([related](https://github.com/openai/whisper/discussions/264)), but I think that's not possible via the API alone. I was playing with the text prompt, but it seems to only make the output accuracy worse.

**Comment by scgalvin** [comment]
Works great, Thanks!

**Comment by naman-gupta99** [comment]
Love this!
